core/modules/system/system.install

```diff
@@ -183,6 +183,22 @@ function system_requirements($phase) {
     $requirements['database_charset'] = $charset_requirements;
   }
 
+  // Report jQuery and jQuery UI versions.
+  if ($phase == 'runtime') {
+    $jquery = backdrop_get_library('system', 'jquery');
+    $jquery_ui = backdrop_get_library('system', 'ui');
+    $requirements['jquery'] = array(
+      'title' => $t('jQuery'),
+      'severity' => REQUIREMENT_OK,
+      'value' => $jquery['version'],
+    );
+    $requirements['jquery_ui'] = array(
+      'title' => $t('jQuery UI'),
+      'severity' => REQUIREMENT_OK,
```

Comment:
This will always report `REQUIREMENT_OK` are there circumstances when the requirement is not OK?

What if a user has hacked core and replaced jQuery lib with an older or newer one than ships with backdrop?

Not sure just asking if we can forsee if in this or other circumstances the status page should at least acknowledge that something is afoot?